src/inc/clrconfigvalues.h

```diff
@@ -558,12 +558,15 @@ CONFIG_DWORD_INFO_EX(INTERNAL_JitVNMapSelLimit, W("JitVNMapSelLimit"), 0, "If no
 RETAIL_CONFIG_DWORD_INFO(INTERNAL_JitVNMapSelBudget, W("JitVNMapSelBudget"), 100, "Max # of MapSelect's considered for a particular top-level invocation.")
 #if defined(_TARGET_AMD64_) || defined(_TARGET_X86_)
 #define EXTERNAL_FeatureSIMD_Default 1
+#define EXTERNAL_JitEnableSSE3_4_Default 1
 #define EXTERNAL_JitEnableAVX_Default 1
 #else // !defined(_TARGET_AMD64_) && !defined(_TARGET_X86_)
 #define EXTERNAL_FeatureSIMD_Default 0
+#define EXTERNAL_JitEnableSSE3_4_Default 0
 #define EXTERNAL_JitEnableAVX_Default 0
 #endif // !defined(_TARGET_AMD64_) && !defined(_TARGET_X86_)
 RETAIL_CONFIG_DWORD_INFO_EX(EXTERNAL_FeatureSIMD, W("FeatureSIMD"), EXTERNAL_FeatureSIMD_Default, "Enable SIMD support with companion SIMDVector.dll", CLRConfig::REGUTIL_default)
+RETAIL_CONFIG_DWORD_INFO_EX(EXTERNAL_EnableSSE3_4, W("EnableSSE3_4"), EXTERNAL_JitEnableSSE3_4_Default, "Enable SSE3, SSSE3, and SSE4 instruction set for wide operations as default", CLRConfig::REGUTIL_default)
```

Comment:
@sivarv I see, but [clr-complus-conf-docgen.sh](https://github.com/dotnet/coreclr/blob/master/Documentation/project-docs/clr-complus-conf-docgen.sh) reads `clrconfigvalues.h` to generate the doc. Do you think we should change `clr-complus-conf-docgen.sh` to generate the doc from `clrconfigvalues.h` and `jitconfigvalues.h` both, and remove jit specific config knobs from `clrconfigvalues.h`?